fsdb.go

```diff
@@ -43,19 +43,19 @@ func NewFSDB(dir string) *FSDB {
 	return database
 }
 
-func (db *FSDB) Get(key []byte) []byte {
```

Comment:
Do we want to keep this DB around, I cant see a reason someone would want to use it?

Reply:
For testing maybe?

Reply:
we can keep it around for now, but should look into cleaning up things that are incomplete(meaning panic("not implemented) is used in places ) or not being used